src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
My apologies I missed how the diff changed, so now we *do* still swap in the ForceVersionTagger if you specify --use-version, it's just a passthrough straight to VersionTagger? This looks to avoid what I was worried about, thanks!